fullscreen-api/main.css

```diff
@@ -0,0 +1,9 @@
+#contents {
+	width: 600px;
```

Comment:
Declaration not really needed?

Reply:
Yeah, I used Picture in picture example as a base and left a lot of things like these untouched. Updated now.